app/assets/stylesheets/views/comments.scss

```diff
@@ -269,3 +269,71 @@
     color: var(--base-90);
   }
 }
+
+.response-templates-container header {
+  padding-bottom: 1rem;
+}
+.personal-template-button,
+.moderator-template-button {
+        --border-width: 0px;
```

Comment:
Nitpick: the indentation seems off here.

Reply:
Resolved! Thanks